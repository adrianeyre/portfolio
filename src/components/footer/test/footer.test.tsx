/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';

import Footer from '../footer';

describe('Footer', () => {
	it('Should render correctly', () => {
		const footerComponent = render(<Footer />);

		expect(footerComponent).toMatchSnapshot();
	});
});
