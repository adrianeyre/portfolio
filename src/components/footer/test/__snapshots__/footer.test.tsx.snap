// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render correctly 1`] = `ShallowWrapper {}`;
